css/src/tokens/palette.scss

```diff
@@ -137,6 +161,11 @@ $palette-green-100: #061a00 !default;
 $palette-green-opacity-30: hsl(120deg 37% 45% / 30%) !default;
 $palette-green-opacity-70: hsl(120deg 37% 45% / 70%) !default;
 
+$palette-green-10-fluent: #f1faf1 !default;
```

Comment:
Are there really only 4 greens in the fluent palette?

Reply:
There are more, but these are the only ones that I see actively being used. I can add the rest.